aiida/engine/utils.py

```diff
@@ -180,13 +180,15 @@ def exponential_backoff_retry(fct, initial_interval=10.0, max_attempts=5, logger
             if ignore_exceptions is not None and isinstance(exception, ignore_exceptions):
```

Comment:
ignore_exceptions seems to be missing a docstring. Can you add one?
It's not self-evident that "ignoring exceptions" means re-rasing them

Reply:
Done